go.mod

```diff
@@ -2,12 +2,13 @@ module github.com/edgexfoundry/go-mod-bootstrap
 
 require (
 	github.com/BurntSushi/toml v0.3.1
-	github.com/edgexfoundry/go-mod-configuration v0.0.0
+	github.com/edgexfoundry/go-mod-configuration v0.0.3
```

Comment:
Lenny - curious, why did we jump from 0.0.0 to 0.0.3 on version?

Reply:
@jpwhitemn , versions 1 & 2 where change to Go 1.13 & PR template, so no functionality changes.